lib/studies/reply/adaptive_nav.dart

```diff
@@ -780,6 +780,22 @@ class _BottomAppBarActionItems extends StatelessWidget {
     return Consumer<EmailStore>(
       builder: (context, model, child) {
         final onMailView = model.onMailView;
+        Color starIconColor;
+
+        if (onMailView) {
+          var currentEmailStarred = false;
+
+          if (model.emails[model.currentlySelectedInbox].isNotEmpty) {
+            currentEmailStarred = model.isEmailStarred(
+              model.emails[model.currentlySelectedInbox]
+                  .elementAt(model.currentlySelectedEmailId),
```

Comment:
Could this instead be backed by a `Set`, and then use `.contains()`?

Reply:
It can! I'll explore changing the implementation to a set in another PR.